test/functional/lua/fs_spec.lua

```diff
@@ -308,26 +309,122 @@ describe('vim.fs', function()
       )
     end)
 
-    it('works with UNC paths', function()
-      eq('//foo', vim.fs.normalize('//foo')) -- UNC path
-      eq('//foo/bar', vim.fs.normalize('//foo//bar////')) -- UNC path
-      eq('/foo', vim.fs.normalize('///foo')) -- Not a UNC path
-      eq('/', vim.fs.normalize('//')) -- Not a UNC path
-      eq('/', vim.fs.normalize('///')) -- Not a UNC path
-      eq('/foo/bar', vim.fs.normalize('/foo//bar////')) -- Not a UNC path
-    end)
-
-    if is_os('win') then
+    if not is_os('win') then
```

Comment:
Why is a "not windows" check needed? Are the following not true for UNC/windows paths as well?

Reply:
Windows and POSIX paths are processed differently, trying to get both Windows and POSIX paths to work simultaneously in both platforms might be a lot more complicated, and the benefit is quite low

Reply:
OK, fair enough.

Reply:
> OK, fair enough.

We could however choose to add an option that allows picking the path type, if this is ever needed. Since processing the path is easy if we know what type of path it is, what's hard is being able to process either type of path simultaneously without knowing which one it is